Neos.ContentRepository.Core/Classes/Feature/ContentStreamClosing/Command/CloseContentStream.php

```diff
@@ -18,9 +18,8 @@
 use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
 
 /**
- * @internal implementation detail. You must not use this command directly.
- * Direct use may lead to hard to revert senseless state in your content repository.
- * Please use the higher level workspace commands instead.
+ * @internal only exposed for testing purposes. You must not use this command directly.
+ * Direct use will leave your content stream in a closed state.
```

Comment:
What about changing the affected tests to work on the event instead and get rid of the command?

Reply:
Yes but that is a followup on this id say to have it decoupled.